Tests/TuistGeneratorTests/ProjectMappers/SynthesizedResourceInterfaceProjectMapperTests.swift

```diff
@@ -48,8 +48,8 @@ final class SynthesizedResourceInterfaceProjectMapperTests: TuistUnitTestCase {
         let aAsset = aAssets.appending(component: "asset")
         let frenchStrings = targetAPath.appending(components: "fr.lproj", "aStrings.strings")
         let frenchStringsDict = targetAPath.appending(components: "fr.lproj", "aStrings.stringsdict")
-        let englishStrings = targetAPath.appending(components: "en.lproj", "aStrings.strings")
-        let englishStringsDict = targetAPath.appending(components: "en.lproj", "aStrings.stringsdict")
+        let englishStrings = targetAPath.appending(components: "en.lproj", "aStringsEn.strings")
```

Comment:
Is there a reason why you added `En` suffix? AFAIK it's general practice to name the strings file the same and differentiate the different language by the directory they are in

Reply:
if it was added to assert the `contents` of `FileDescriptor`s, I'd rather change the method stub to include its parent directory in the addition of its basename

Reply:
Yeah, sorry it was something I used for a quick test and forgot to fix it in the end, fixing it now